src/app/pages/profile/profile.component.html

```diff
@@ -0,0 +1,3 @@
+<ngx-general-information [user]='user'></ngx-general-information>
+<ngx-contact-information></ngx-contact-information>
```

Comment:
We should pass the user to the `ContactInformationComponent` as well. At the very least to display the correct email address.

Reply:
@sentenza the User type only contains a name and a picture property at the moment.  

Would you like me to populate the User with the relevant additional properties to satisfy the details in the profile page?

Reply:
Yes, it would be awesome! 😃

Reply:
'That's true, the User type in @core/data should be refactored in order to hold at least the essential information like the email address. Anyway, on this matter, I would wait for the maintainer's feedback.'